llvm/test/CodeGen/SPIRV/structurizer/cf.switch.ifstmt.simple2.ll

```diff
@@ -20,21 +19,21 @@
 ;       b += 2;
 ;       break;
 ;     case 3:
-;     {
-;       b += 3;
-;       break;
-;     }
+;       {
+;         b += 3;
+;         break;
+;       }
 ;     case t:
 ;       b += t;
 ;     case 4:
 ;     case 5:
 ;       b += 5;
 ;       break;
 ;     case 6: {
-;     case 7:
-;       break;}
+;               case 7:
+;                 break;}
 ;     default:
-;       break;
+;             break;
```

Comment:
Was this change intended?

Reply:
no, error when doing a clang-format of the code. Reverted this part.

Reply:
Other PR had the same change. This PR will revert those format changes.